dev-packages/env-tests/cloudflare-astro/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@sentry-internal/env-tests-cloudflare-astro",
+  "type": "module",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "yarn build:bundle",
+    "build:bundle": "astro build",
+    "preview": "astro preview",
+    "astro": "astro"
+  },
+  "dependencies": {
+    "astro": "^4.1.1",
+    "@astrojs/cloudflare": "^8.0.2",
+    "@sentry/astro": "7.92.0",
+    "@sentry/profiling-node": "^1.3.2"
```

Comment:
Does this take the local versions of the package?

Reply:
If I understand correctly: there isn't any local version of profiling-node, since it is not in the sentry-javascript repository.

Reply:
Right. So what I'm thinking here is that if we take already published packages, we'll only know in hindsight if something broke. Which is better than nothing ofc but we worked around this in our e2e tests. 

For `@sentry/astro`, we could take the same approach as in our e2e tests and use the local verdaccio repository we set up when initializing the e2e tests. 

For profiling-node we'll have to use the published version as long as it's not in the monorepo.

Reply:
Or IOW, maybe moving this to the e2e tests is beneficial here for the verdaccio stuff. FWIW, not all e2e test apps also actually _test_ things. Sometimes we just use them to check that building works correctly.

Reply:
Yup, I moved to e2e-tests